internal/clients/diffreviewer/github/github_service.go

```diff
@@ -176,6 +169,15 @@ func (s *Service) LoadConfig(nightfallConfigFileName string) (*nightfallconfig.C
 	if s.CheckRequest.SHA == "" {
 		s.CheckRequest.SHA = event.HeadCommit.ID
 	}
+	baseRev, ok := os.LookupEnv(BaseRefEnvVar)
+	if !ok || baseRev == "" {
+		s.Logger.Debug(fmt.Sprintf("Could not get environment variable %s so using Before from the event object", BaseRefEnvVar))
+		baseRev = event.Before
```

Comment:
let's log an error if `event.Before` is empty unless we're guaranteed to have it. On a side note , do you know what would happen for initial commits to new repos?

Reply:
I don't think we want an error because if `BaseRefEnvVar` is empty we can still get the `Before` commit hash. This happens for when the action is run on a push.